### PR TITLE
Fixed pages showing up in wrong language

### PR DESCRIPTION
**Pull request by JackieNim**
Fix issue where links show up in the wrong language
https://github.com/iluwatar/java-design-patterns-web/issues/22

### CODE REVIEW

**Review by ohbus — CHANGES_REQUESTED** [review]
Kindly take a look at the comments.

File: fr/README.md [hunk 1]
```diff
@@ -15,56 +15,46 @@
...
-notre [wiki développeur](https://github.com/iluwatar/java-design-patterns/wiki).
-Nous vous aiderons répondrons à vos questions dans le [Gitter chatroom](https://gitter.im/iluwatar/java-design-patterns).
+notre [wiki développeur](https://github.com/iluwatar/java-design-patterns/wiki). Nous aiderons
+vous et répondez à vos questions dans le [Gitter chatroom](https://gitter.im/iluwatar/java-design-patterns).
 
 # Licence
 
-Ce projet est concédé sous les termes de la licence MIT.
\ No newline at end of file
+Ce projet est concédé sous les termes de la licence MIT.
```
[the remark this answers is not in the document]
Comment (JackieNim) [reply fr/README.md]: I think there was a problem when I tired forking the project 🤔 Let me try again

[the remark this answers is not in the document]
Comment (JackieNim) [reply fr/README.md]: @ohbus https://github.com/iluwatar/java-design-patterns/pull/1752 is the new PR

**Comment by iluwatar** [comment]
> @iluwatar We may need to update the Wiki for the same changes we will be proposing to help new contributors and old ones as well to get familiarized with the new standard of adding language selection.

Good point @ohbus. I created a new issue https://github.com/iluwatar/java-design-patterns/issues/1755 about this.

**Comment by ohbus** [comment]
@iluwatar Shall we reopen #1752 against this and re-review the whole doc again?

Or is there any way that you know where @JackieNim can rrecoverthis deleted fork to make [this PR](https://github.com/iluwatar/java-design-patterns/pull/1750) open again?

**Comment by iluwatar** [comment]
Since the repository has been deleted I don't know any other way than reopening #1752
